Model/Config.php

```diff
@@ -286,23 +295,25 @@ public function isActivated($scopeId = null, $scope = null)
      * @param  string|null                     $secret
      * @param  boolean|int|null                $isEnabled
      * @param  int|null                        $storeId
+     * @param  string|null                     $scopes
      * @return $this
      */
-    public function setStoreCredentialsAndIsEnabled($appKey, $secret, $isEnabled, $storeId = null)
+    public function setStoreCredentialsAndIsEnabled($appKey, $secret, $isEnabled, $storeId = null, $scopes = ScopeInterface::SCOPE_STORES)
     {
-        $this->resourceConfig->saveConfig(self::XML_PATH_YOTPO_APP_KEY, $appKey, ScopeInterface::SCOPE_STORES, $storeId ?: $this->storeManager->getStore()->getId());
-        $this->resourceConfig->saveConfig(self::XML_PATH_YOTPO_SECRET, ($secret ? $this->encryptor->encrypt($secret) : null), ScopeInterface::SCOPE_STORES, $storeId ?: $this->storeManager->getStore()->getId());
-        $this->resourceConfig->saveConfig(self::XML_PATH_YOTPO_ENABLED, $isEnabled, ScopeInterface::SCOPE_STORES, $storeId ?: $this->storeManager->getStore()->getId());
+        $this->resourceConfig->saveConfig(self::XML_PATH_YOTPO_APP_KEY, $appKey, $scopes, is_null($storeId) ? $this->storeManager->getStore()->getId() : $storeId);
```

Comment:
please do not use one liners. 
change the expression:
   is_null($storeId) ? $this->storeManager->getStore()->getId() : $storeId
to:

if (is_null($storeId))
    $storeId  = $this->storeManager->getStore()->getId();

please put the above expression in the beginning of the function instead of embedding it inside the saveConfig function.

Reply:
ok